Bulldozer.CSV/CSVComponent.cs

```diff
@@ -708,7 +708,7 @@ private static CSVInstance.RockDataType GetRecordTypeFromFilename( string filena
          * Definition for the Batch.csv import file:
          */
 
-        private const int BatchID = 0;
+        private const int BatchId = 0;
```

Comment:
@treyhendon does this change require an update to the sample CSV for Batches? I was not sure if the column in the csv is case sensitive or not, https://github.com/KingdomFirst/Bulldozer/blob/c3b69615de3cf4baec2daa477b38080ac33f4d2e/Bulldozer.CSV/SampleCSVs/04%20-%20Batch.csv

Reply:
Good call for consistency.  The CSV reader in Bulldozer doesn't use the file header names, but it's nice to keep them the same for reference.

These constants in the Component file become the zero-index legend in the CSV maps for referencing specific columns.  So that way, in the financial map, rather than saying `var batchIdKey = row[0];` we can use `row[BatchID];` for readability.